src/main/webapp/app/admin/logs/logs.component.ts

```diff
@@ -69,6 +81,7 @@ export class LogsComponent implements OnInit, OnDestroy {
 
     ngOnDestroy() {
         // prevent memory leak when component destroyed
-        this.subscription.unsubscribe();
+        this.activeRouteSubscription.unsubscribe();
```

Comment:
instead of keeping track to unsubscribe, can you use `takeUntil` operator like done in the `main.component.ts` for oauth2 flow?